tests/e2e/export-dfx-identity.test.js

```diff
@@ -0,0 +1,186 @@
+  await closeButton.click();
+};
+
+const isContinueButtonDisabled = async (page) => {
+  const isContinueButtonDisabledFunc = await page.$('[data-testid="continue-button"][disabled]') !== null;
+  expect(isContinueButtonDisabledFunc).toBe(true);
+};
+
+const checkIsSubtitleTextMatch = async (page) => {
+  const subtitle = await page.getByTestId('subtitle', true);
+  const subtitleText = await page.evaluate((el) => el.textContent, subtitle);
+  expect(subtitleText).toBe('In order to use your account in DFX, run the following command, using the .pem file download as route.');
+};
+
+const copyCommandIconButtonClick = async (page) => {
+  const copyCommandIconButton = await page.getByTestId('copy-button', true);
+  await copyCommandIconButton.click();
+};
+
+const accountSelectClick = async (page) => {
+  const accountSelect = await page.getByTestId('account-select', true);
+  await accountSelect.click();
+};
+
+const accountItemClick = async (page) => {
+  const accountItem = await page.getByTestId('select-item-Subaccount', true);
+  await accountItem.click();
+  await page.waitForTestIdSelector('select-item-Subaccount', { hidden: true });
+};
+
+const checkIsAccountNameMatch = async (page) => {
+  const accountName = await page.getByTestId('account-name', true);
+  const accountNameText = await page.evaluate((el) => el.textContent, accountName);
+  expect(accountNameText).toBe('Subaccount');
+};
+
+describe('Settings: Export DFX Identity', () => {
+  let browser;
+  let page;
+  const wrongPassword = '1234544444';
+
+  beforeAll(async () => {
+    browser = await setupChrome();
+
+    // Importing the account
+    page = await utils.createNewPage(browser);
+    await optionsPageUtils.importAccount(page, secrets.seedphrase, secrets.password);
+    await optionsPageUtils.unlock(page, secrets.password);
+
+    await page.close();
+  });
+
+  beforeEach(async () => {
+    page = await utils.createNewPage(browser);
+    await page.goto(chromeData.popupUrl);
+    await popupPageUtils.profileButtonClick(page);
+    await settingsButtonClick(page);
+    await exportDFXIdentityButtonClick(page);
+  });
+
+  afterEach(async () => {
+    await page.close();
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  test('checking the continue button is disabled when password input is empty', async () => {
+    await safeCheckboxClick(page);
+    await continueButtonClick(page);
+    await isContinueButtonDisabled(page);
+  });
+
+  test('checking that password asking for the second time, when enter password view was leaved', async () => {
+    await backButtonClick(page);
+    await exportDFXIdentityButtonClick(page);
+    await isContinueButtonDisabled(page);
+  });
+
+  test('checking that close button taken user to main view', async () => {
+    await closeButtonClick(page);
+    await page.waitForSelector('[data-testid="open-send-view-button"]');
+  });
+
+  test('entering wrong password', async () => {
```

Comment:
I was thinking of using the same working for all the tests like ("checking user cannot see the DFX identity when password is incorrect")

Reply:
Also, I noticed that when the password is incorrect, and user tries to go into Plug ext, it asks you for the password again to plug, probably we want to kind of complete that flow and make sure that is indeed happening, that happens for security reasons.

Reply:
> Also, I noticed that when the password is incorrect, and user tries to go into Plug ext, it asks you for the password again to plug, probably we want to kind of complete that flow and make sure that is indeed happening, that happens for security reasons.

> Rn, it's a bug with locking. The wallet locks randomly, that it's why I should run my tests separately. Should be fixed soon :)